src/parallel-rustc.md

```diff
@@ -1,31 +1,35 @@
 # Parallel Compilation
 
-As of <!-- date-check --> August 2022, the only stage of the compiler that 
-is already parallel is codegen. Some parts of the compiler already have 
-parallel implementations, such as query evaluation, type check and 
-monomorphization, but the general version of the compiler does not include 
-these parallelization functions. **To try out the current parallel compiler**, 
-one can install rustc from source code with `parallel-compiler = true` in 
-the `config.toml`.
+As of <!-- date-check --> August 2022, the only stage of the compiler that is
+parallel is [code generation stage][codegen] (codegen). Some parts of the
+compiler have parallel implementations, such as query evaluation, type check
+and [monomorphization][monomorphization], but the general version of the
+compiler does not include parallelization functions. **To try out the current
+parallel compiler**, install `rustc` from source code with `parallel-compiler =
+true` in the `Config.toml`.
```

Comment:
Most of the rust compilers are now parallelized.

The codegen part is executed concurrently by default. You can use the `-C codegen-units=n` option to control the number of concurrent tasks.

The parts after HIR lowering to codegen such as type checking, borrowing checking, and mir optimization are parallelized in the nightly version. Currently, they are executed in serial by default, and parallelization is manually enabled by the user using the `-Z threads = n` option.

Other parts, such as lexical parsing, HIR lowering, and macro expansion, are still executed in serial mode.

Reply:
I'll slightly adjust this to reflect your wording and then back link to the issue to update the docs for parallel front-end.